### ISSUE TITLE
voice_members not updating

### ISSUE THREAD

**Issue by tuckismad** [Rapptz/discord.py#37]
I use this code to list all voice members in each channel on my server
```python
server = discord.utils.find(lambda m: m.id == self.server, self.client.servers)
                if server is not None:
                    for channel in server.channels:
                        print(channel.voice_members)
```

However if i move channels after my bot has connected to the server, voice_members doesn't reflect the changes. it still shows me in the old channel.

Hopefully i'm just missing something small but i've been at this for a while so i believe its a bug.

**Comment by tuckismad** [comment]
I've managed to fix this however my git isn't playing nicely. Will push my commit shortly.

**Comment by tuckismad** [comment]
Fixed in pull request #38

**Comment by tuckismad** [comment]
Fixed in pull request #43
